addons/auth_ldap/users_ldap.py

```diff
@@ -69,13 +75,16 @@ def connect(self, conf):
         :return: an LDAP object
         """
 
-        uri = 'ldap://%s:%d' % (conf['ldap_server'],
+        if conf['ldap_tls']:
```

Comment:
Shouldn't that be 'ldap_ssl'? TLS is established via a 'ldap' (plaintext) connection, whereas 'ldaps' requires a full SSL connection, if I understand it correctly.